### PR TITLE
Improved the explanation about the verbosity levels of the console

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: components/console/introduction.rst [hunk 1]
```diff
@@ -173,22 +173,18 @@ Verbosity Levels
    The ``VERBOSITY_VERY_VERBOSE`` and ``VERBOSITY_DEBUG`` constants were introduced
    in version 2.3
 
-The console has 5 levels of verbosity. These are defined in the
+The console has five levels of verbosity. These are defined in the
```
Comment (cordoval) [inline components/console/introduction.rst]: five verbosity levels.

**Comment by xabbuh** [comment]
Is this related to #5443?

File: components/console/introduction.rst [hunk 2]
```diff
@@ -173,22 +173,18 @@ Verbosity Levels
...
-level of verbosity.
+===========================================  ==================================  =====================
+Value                                        Meaning                             Console option
+===========================================  ==================================  =====================
+``OutputInterface::VERBOSITY_QUIET``         Do not output any messages          ``-q`` or ``--quiet``
+``OutputInterface::VERBOSITY_NORMAL``        The default verbosity level         (none)
+``OutputInterface::VERBOSITY_VERBOSE``       Increased verbosity of messages     ``-v``
+``OutputInterface::VERBOSITY_VERY_VERBOSE``  Informative non essential messages  ``-vv``
+``OutputInterface::VERBOSITY_DEBUG``         Debug messages                      ``-vvv``
+===========================================  ==================================  ======================
```
Comment (xabbuh) [inline components/console/introduction.rst]: there's an additional `=`

File: components/console/introduction.rst [hunk 3]
```diff
@@ -198,7 +194,7 @@ level of verbosity.
 It is possible to print a message in a command for only a specific verbosity
 level. For example::
 
-    if (OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity()) {
+    if ($output->getVerbosity() >= OutputInterface::VERBOSITY_VERBOSE) {
```
Comment (WouterJ) [inline components/console/introduction.rst]: this is already done in #5381

**Comment by javiereguiluz** [comment]
I've made all the required changes. If you agree with it, we can label it as `finished`. Thanks.

**Comment by xabbuh** [comment]
:+1: reads well

**Comment by weaverryan** [comment]
This is *wonderful*. Thanks Javier!

**Pull request merged**